_local/platform/darwin.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+### BEGIN Platform Specific Section
+
+function preDockerCommands() {
+    local tag=$1
+    # Docker Desktop for Darwin is stupid, and doesn't read FROM image
+    # Also fails with confusing "denied: request access" in addition to error
+    # when build fails, instead of just error.
+    if [ "$(docker image ls -q --filter "reference=golang:$tag")" == "" ]; then
+        docker image pull "golang:$tag"
+    fi
+}
```

Comment:
If this is the main difference between darwin and linux targets, I would probably do that always instead of having to keep platform specific logic.